src/bin/efreet/efreet_mime_cache_create.c

```diff
@@ -373,7 +373,7 @@ store_cache(const char *out)
    ERR("Cannot write to %s", buf);
 error:
    if (f) fclose(f);
-   if (unlink(buf) != 0) WRN("Cannot delete tmp file %s", buf);
+   if (eina_file_unlink(buf) != 0) WRN("Cannot delete tmp file %s", buf);
```

Comment:
Probably you'll want to change to `!= EINA_TRUE`.

Reply:
```suggestion
   if (eina_file_unlink(buf) != EINA_TRUE) WRN("Cannot delete tmp file %s", buf);
```